controllers/custom/builder.go

```diff
@@ -130,24 +130,30 @@ func (b *Builder) Complete(reconciler Reconciler) (healthz.Checker, error) {
 				if err != nil {
 					return err
 				}
+				metaObj, ok := convertedObj.(metav1.Object)
+				if !ok {
+					return fmt.Errorf("failed to get object meta %v", obj)
+				}
+				b.log.Info("processing item", "objName", metaObj.GetName(), "namespace", metaObj.GetNamespace(), "deltaType", d.Type)
```

Comment:
is there no knob to control verbosity for this log? the volume of this can be huge

Reply:
Added as info since this change will be deployed only to few clusters where we need the logs.

Reply:
How are we planning to remove this? Will we revert the PR once the image is built?

Reply:
This is not being merged into master, it will be merged to `debug-leakedENI` and we'll build the image from this branch. I don't think we would need to revert the PR in this case.

Reply:
Actually the introspect API will be useful for general case, I'll pick the relevant changes to be merged into master in a subsequent PR.